packages/experimental/Shadow/src/shadowStyle.ts

```diff
@@ -0,0 +1,46 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { ColorValue } from 'react-native';
+
+export const shadowStyleFromTheme = (t: Theme, shadowToken: string) => {
+  const keyShadow = t.shadows[shadowToken].key;
+  const ambientShadow = t.shadows[shadowToken].ambient;
+
+  return {
+    key: {
+      // iOS Shadow props
```

Comment:
Thinking I should remove this comment (and the one on line 22), and then was thinking maybe we could move the android shadow props somewhere else? Maybe a different method?

Reply:
Having platform specific files I think would be a good idea. The iOS props (afaik) are applicable to iOS/macOS/win32, so that could stay `shadowStyle.ts`, while Android has a `shadowStyle.android.ts` that simply returns "elevation".

Reply:
Discussed offline but I tried this and had some issues with running the tests by platform, we also decided Android is out of scope for this component